spec/support/idv_helper.rb

```diff
@@ -29,13 +29,21 @@ def fill_out_phone_form_ok(phone = '415-555-0199')
     fill_in :idv_phone_form_phone, with: phone
   end
 
+  def fill_out_phone_form_fail
+    fill_in :idv_phone_form_phone, with: '(555) 555-5555'
+  end
+
+  def click_idv_continue
```

Comment:
minor: I usually expect method calls to occur above method definitions (so would put this below `complete_idv_profile` somewhere)